dynamicprefixvici.c

```diff
@@ -8,35 +8,25 @@
 typedef struct CommandLineArguments
 {
     char* pool_name;
-    char* old_prefix_env;
-    char* new_prefix_env;
+    char* new_prefix;
```

Comment:
new_prefix => prefix

As we have no more distinction between the "old" and "new" prefixes, I think we can just make this variable called "prefix".